recipe/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set name = "recommenders" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/microsoft/recommenders/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8db3785543b2ace6ddc68994e71c8920efa781432708d8d4cc7aff720f11b55f
+
+build:
+  number: 0
+  skip: True  # [py<38 or py>=310 or s390x]
```

Comment:
Seems like recommenders supports more than just python 3.9 but perhaps you are just simplifying.

Reply:
Based on the recommenders tag for 1.1.1, for the [setup.py](https://github.com/microsoft/recommenders/blob/b704c420ee20b67a9d756ddbfdf5c9afd04b576b/setup.py#L130), it seems that recommenders doesn't have python 3.10 compatibility at the moment.